src/main/java/de/unistuttgart/informatik/fius/icge/territory/EntityState.java

```diff
@@ -12,4 +12,24 @@
 
 public interface EntityState {
     public Entity createEntity(Simulation sim);
+
+    /**
+     * Checks wether the entity is considered  a solid entity.
+     * 
+     * @return true if entity is solid
+     */
+    public boolean isSolid();
```

Comment:
Maybe add a default which returns `false` such that an entity can be implemented faster by students.